app/.meteor/packages

```diff
@@ -26,4 +26,7 @@ dapearce:material-icons
 # Our stack
 coffeescript
 mquandalle:jade
+fortawesome:fontawesome
```

Comment:
I actually intended to try out the material icon set for this project. I need to update the wireframes to reflect this. I added the package in a previous PR (`dapearce:material-icons`) http://zavoloklom.github.io/material-design-iconic-font/cheatsheet.html